.changeset/slow-dolls-type.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-scaffolder': minor
+'@backstage/plugin-catalog': patch
+---
+
+- Fixed a bug in DefaultEntityPresentationApi that was not using default context to parse entityOrRef which led to undefined Kind and wrong entityRef when using entityPresentationApi
```

Comment:
If you put technical identifiers in ``` `backticks` ``` they get ignored by the spell checker

Reply:
I'm not sure what you are asking, you want me to put technical identifiers with backticks ?